Panels/tests/Panels-specs.js

```diff
@@ -66,6 +120,53 @@ describe('Panels Specs', () => {
 		expect(panel.id).toBe('p2');
 	});
 
+	test('should set custom autoFocus on child panels', () => {
+		const AutoFocusPanel = ({autoFocus, id}) => <Panel autoFocus={autoFocus} data-testid="panel" id={id}>{autoFocus}</Panel>;
+		render(
+			<Panels index={0}>
+				<AutoFocusPanel autoFocus="focus-here" id="p2" />
+			</Panels>
+		);
+
+		const expected = 'focus-here';
+		const panel = screen.getByTestId('panel');
+
+		expect(panel.textContent).toBe(expected);
+		expect(panel.id).toBe('p2');
+	});
+
+	test('should call \'onBack\' for \'escape\' key if not on first panel', () => {
+		const spy = jest.fn();
+		render(
+			<Panels data-testid="panels" index={1} onBack={spy}>
+				<Panel>First</Panel>
+				<Panel>Second</Panel>
+			</Panels>
+		);
+
+		const panels = screen.getByTestId('panels');
+
+		fireEvent.keyDown(panels, {keyCode: 27});
```

Comment:
Please define these on top of page to be more explicit, like so:
const keyDown = (keyCode) => (elm) => fireEvent.keyDown(elm, {keyCode});
const escapeKeyDown = keyDown(27);

const keyUp = (keyCode) => (elm) => fireEvent.keyUp(elm, {keyCode});
const escapeKeyUp = keyUp(27);